third_party/warp_transducer/binding.cpp

```diff
@@ -0,0 +1,47 @@
+#include <iostream>
+#include <numeric>
+
+#include <torch/extension.h>
```

Comment:
I think `<torch/script.h>` is the appropriate header.

Reply:
(cc https://github.com/pytorch/pytorch/issues/18718#issuecomment-480913493)